api/types/constants.go

```diff
@@ -662,6 +662,11 @@ const (
 	// ReqAnnotationNotifyServicesLabel is the request annotation key at which notify services are stored for access plugins.
 	ReqAnnotationNotifyServicesLabel = "/notify-services"
 
+	// ReqAnnotationApprovedCloseCode is the request annotation key at which the servicenow close code for approved access requests is stored, this label is added to roles to determine what close code to use when closing the ServiceNow incidents created for access requests.
+	ReqAnnotationApprovedCloseCode = "/approved-close-code"
```

Comment:
I still see that you check `req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationDeniedCloseCode]` but I don't see where you set the system annotations at all.
So, I guess that something is still missing somewhere to set this value

Reply:
The current idea for this is that these are set in the requesting role